lib/src/visitor/async_evaluate.dart

```diff
@@ -1323,30 +1387,41 @@ class _EvaluateVisitor
 
   Future<Value> visitIncludeRule(IncludeRule node) async {
     var mixin = _addExceptionSpan(node,
-            () => _environment.getMixin(node.name, namespace: node.namespace))
-        as UserDefinedCallable<AsyncEnvironment>;
+        () => _environment.getMixin(node.name, namespace: node.namespace));
     if (mixin == null) {
       throw _exception("Undefined mixin.", node.span);
     }
 
-    if (node.content != null && !(mixin.declaration as MixinRule).hasContent) {
-      throw _exception("Mixin doesn't accept a content block.", node.span);
-    }
+    if (mixin is AsyncBuiltInCallable) {
+      if (node.content != null) {
+        throw _exception("Mixin doesn't accept a content block.", node.span);
+      }
 
-    var contentCallable = node.content == null
-        ? null
-        : UserDefinedCallable(node.content, _environment.closure());
-    await _runUserDefinedCallable(node.arguments, mixin, node, () async {
-      await _environment.withContent(contentCallable, () async {
-        await _environment.asMixin(() async {
-          for (var statement in mixin.declaration.children) {
-            await statement.accept(this);
-          }
+      await _runBuiltInCallable(node.arguments, mixin, node);
+    } else if (mixin is UserDefinedCallable<AsyncEnvironment>) {
+      if (node.content != null &&
+          !(mixin.declaration as MixinRule).hasContent) {
+        throw _exception("Mixin doesn't accept a content block.", node.span);
+      }
+
+      var contentCallable = node.content == null
+          ? null
+          : UserDefinedCallable(node.content, _environment.closure());
+
+      await _runUserDefinedCallable(node.arguments, mixin, node, () async {
+        await _environment.withContent(contentCallable, () async {
+          await _environment.asMixin(() async {
+            for (var statement in mixin.declaration.children) {
+              await statement.accept(this);
+            }
+          });
+          return null;
         });
         return null;
       });
-      return null;
-    });
+    } else {
+      throw UnsupportedError("Unknown callable type $mixin.");
```

Comment:
Did you mean to print the `.runtimeType` here?

Reply:
I was just planning on relying on the object's toString(). I thought `Object.runtimeType` was in the language team's crosshairs.